community/louvain_common.go

```diff
@@ -35,11 +36,194 @@ func Q(g graph.Graph, communities [][]graph.Node, resolution float64) float64 {
 	}
 }
 
+// ReducedGraph is a modularised graph.
+type ReducedGraph interface {
+	graph.Graph
+
+	// Communities returns the community memberships
+	// of the nodes in the graph used to generate
+	// the reduced graph.
+	Communities() [][]graph.Node
+
+	// Structure returns the community structure of
+	// the current level of the module clustering.
+	// The first index of the returned value
```

Comment:
Ahhh! Sorry, it should be lower - the level with more nodes. You can see this from the implementation of Communities methods.

How about "The first index of the returned value corresponds to the index of the nodes in the ReducedGraph returned by Expanded if it is non-nil."